cmd/minishift/cmd/start.go

```diff
@@ -779,3 +784,17 @@ func cacheMinishiftISO(config *cluster.MachineConfig) {
 func shouldPreflightChecksBeSkipped() bool {
 	return viper.GetBool(configCmd.SkipPreflightChecks.Name)
 }
+
+func getOpenShiftVersion() (string, error) {
+	tag := viper.GetString(configCmd.OpenshiftVersion.Name)
+	if tag == "vlatest" {
+		tags, err := openshiftVersion.GetSortedOpenShiftTags(constants.MinimumSupportedOpenShiftVersion, version.GetOpenShiftVersion())
+		if err != nil {
+			return "", nil
```

Comment:
This is wrong, you are supposed to return err here but returning nil that means this function never return the err now.

Reply:
Ya missed that thanks

Reply:
Done

Reply:
still seeing `return "", nil` ?